### DISCUSSION TITLE
Audio Streaming- Fairseq version [Help Needed]

### DISCUSSION THREAD

**Discussion by HardikJain02** [coqui-ai/TTS#3768] [General Q&A]
Hi Community!

1. **Current Achievement:** I've achieved real-time audio streaming with a latency of around 200ms for my use case.
2. **Language Issue:** However, the current language support has only 17 languages. I am targeting for a lot more, I'm considering using Fairseq as a solution
3. **Question:** Can Fairseq be implemented for real-time TTS with these two codes? [[Streaming Inference]](https://docs.coqui.ai/en/dev/models/xtts.html#streaming-manually) [[Fairseq Implementation]](https://docs.coqui.ai/en/latest/inference.html#example-text-to-speech-using-fairseq-models-in-1100-languages)
4. **Request for Help:** Any advice on using Fairseq or anything else for this specific use case would be greatly appreciated.

**Comment by oscody** [comment]
Hey can you talk about how you achieved real-time audio streaming ?
